app/Actions/ValidateGithubConfiguration.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Actions;
+
+use App\ConsoleWriter;
+use App\Shell;
+use Symfony\Component\Process\ExecutableFinder;
+
+class ValidateGithubConfiguration
+{
+
+    public const WARNING_UNABLE_TO_CREATE_REPOSITORY = 'Unable to create a new GitHub repository';
+    public const INSTRUCTIONS_GH_NOT_INSTALLED = [
+        "Lambo uses the official GitHub command line tool to create new repositories but it doesn't seem to be installed.",
```

Comment:
Hm, does Laravel use `gh`? I've always used `hub`. lmk what you think!

https://github.com/github/hub

Reply:
Yes, Laravel uses _gh_. I've never used _hub_ so I don't have an opinion on the tool its self. 

Having looked into _hub_ a little, there are philosophical differences between the two tools, _hub_ wraps git where as _gh_ uses the GitHub API and only provides GitHub workflows. None of this really matters in our case though.

I think I can support both pretty easily so I will add _hub_ support.

Reply:
Added hub support.